### DISCUSSION TITLE
how to create global custom filter and computed property in nuxt 3

### DISCUSSION THREAD

**Discussion by shahensaifullah** [nuxt/nuxt#17492] [Questions]
in nuxt2, i normally create plugins name mixin.js like this

```
plugins/mixins.js

import Vue from 'vue'
import {mapGetters} from "vuex"
const User = {
  install(Vue, option) {
    Vue.filter('capfirst', function (value) {
      if (!value) return ''
      value = value.toString()
      return value.charAt(0).toUpperCase() + value.slice(1)
    })
    Vue.mixin({
      computed: {
        ...mapGetters({
          user: 'userGet',
          auth: 'userCheck'
        })
      }
    })
  }
};
Vue.use(User);
```
Here I create a filter where it always capitalizes the first letter of the word. The second one is global, user identity where I can easily control user whole nuxt page and check if the user auth or not and if auth, what is user identity. but nuxt 3 I cannot create this in plugins. 
nuxt 3 says, Vue.use cannot find as well as Vue.filter and mixin.
So, how can I use two property in my nuxt 3 project. Kindly someone rearranges my code with folder structure.

**Comment by wobsoriano** [comment]
https://v3.nuxtjs.org/guide/directory-structure/plugins#vue-plugins

Also, forget about mixins
